### PR TITLE
Remove hardcoded scripts from layout

### PR DESCRIPTION
**Pull request by erikmueller**
* add example for including an actual script tag in the head

Originally I wanted to add another config to include custom scripts (like the head template) in order to not block site rendering. However as [Google](https://developers.google.com/speed/docs/insights/BlockingJS) is suggesting nowadays yuo should go back to put them into the head (for earlier parsing) and mark them as `async` (for making them non-blocking). 

TL;DR: Just removed the static `<script>` tag and `require` statement and added some docs

### CODE REVIEW

**Comment by erikmueller** [comment]
Or was this issue already addressed?

**Comment by smpallen99** [comment]
Thanks!!
